### PR TITLE
Simple grid

### PR DESCRIPTION
**Pull request by anandkumarcoder**
working link - https://component-library-qh4b9zkpq-anandkumarcoder.vercel.app/docs/docs.html
Feat: grid added
image
image

### CODE REVIEW

**Review by darshandamre — COMMENTED** [review]
looks good to me

**Review by darshandamre — APPROVED** [review]

File: Styles/simple-grid.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.grid-two-col{
+    display: grid;
+    gap: 2rem;
+    grid-template-columns:  repeat(2 , 1fr);
+    width: 100%;
+    
+}
+
+.col-1 , .col-2 , .col-3{
```
Comment (yashpatel1797) [inline Styles/simple-grid.css:9]: If you want the text in the centre then add text-align: centre it will look good.

File: docs/docs.html [hunk 2]
```diff
@@ -673,6 +675,65 @@ <h4>Example of icon buttons</h4>
...
+        </div>
+
+        <div class="code-snippets">
+          <pre><code class="html"> &lt;div class=&quot;code-snippets&quot;&gt;
+            &lt;pre&gt;&lt;code class=&quot;html&quot;&gt;  &amp;lt;div class=&amp;quot;grid-two-col&amp;quot;&amp;gt;
+              &amp;lt;div class=&amp;quot;col-1&amp;quot;&amp;gt; col 1&amp;lt;/div&amp;gt;
+              &amp;lt;div class=&amp;quot;col-2&amp;quot;&amp;gt; col 2&amp;lt;/div&amp;gt;
+              &amp;lt;/div&amp;gt;
+              &lt;/code&gt;&lt;/pre&gt;
+          &lt;/div&gt;
```
Comment (yashpatel1797) [inline docs/docs.html]: Check code snippet in the live link it is showing some extra things.

**Review by yashpatel1797 — CHANGES_REQUESTED** [review]
The code looks good to me. added 2 comments you can check that.

**Comment by anandkumarcoder** [comment]
Feat:simple grid

**Pull request merged**